src/Controls/src/Core/Editor/Editor.iOS.cs

```diff
@@ -9,6 +9,12 @@ public static void MapText(EditorHandler handler, Editor editor) =>
 		public static void MapText(IEditorHandler handler, Editor editor)
 		{
 			Platform.TextExtensions.UpdateText(handler.PlatformView, editor);
+
+			// Any text changes in the editor field require recalculating the CharacterSpacing by regenerating the attributed string to properly apply the spacing and override the current text formatting.
+			if (editor.CharacterSpacing > 0)
```

Comment:
Can invalidate CharacterSpacing just doing:

`handler?.UpdateValue(nameof(CharacterSpacing));`

Reply:
Thanks for the suggestion! I've updated the code as per your recommendation, using handler?.UpdateValue(nameof(CharacterSpacing));